ext/native/gfx_es2/draw_text.cpp

```diff
@@ -574,3 +577,11 @@ void TextDrawer::OncePerFrame() {
 		}
 	}
 }
+
+float TextDrawer::CalculateDPIScale() {
+	float scale = g_dpi_scale;
+	if (scale >= 1.0f) {
```

Comment:
Well, I was originally planning to add a comment, and then decided it wasn't necessary.  I feel like using `std::min` would be less clear here, but maybe it just still deserves a comment.

-[Unknown]